mozilla/donate-wagtail
# minimum amount message now rendering and updating

Closes #1634

**Link to test page**: https://donate-wagta-other-amou-ejbati.herokuapp.com/en-US/

**Steps to test**:

1. Visit the test page linked above
2. Note the new "Minimum amount" message below the "other" value field.
<img width="531" alt="Screen Shot 2022-02-03 at 3 24 14 PM" src="https://user-images.githubusercontent.com/18314510/152446630-6e52a3d9-d0c1-41ed-bc04-2f9720534d05.png">

3. If you type in a number below the minimum amount and click away from the input, a error should pop up below.
<img width="531" alt="Screen Shot 2022-02-03 at 3 25 26 PM" src="https://user-images.githubusercontent.com/18314510/152446574-c6be482c-2da4-4300-8aed-19f578d12fda.png">

4. If you type a number above the minimum amount and click away from the input, the error should disappear.
5. Please select a new currency from the drop down at the top right.
<img width="531" alt="Screen Shot 2022-02-03 at 3 24 14 PM" src="https://user-images.githubusercontent.com/18314510/152446316-f753b45f-6552-44f2-9d58-1f0e3234fcf5.png">

6. Once selected, the form should update, and the minimum amount message should also update with a new currency.
<img width="531" alt="Screen Shot 2022-02-03 at 3 26 42 PM" src="https://user-images.githubusercontent.com/18314510/152446513-28815765-abaf-40e7-853f-edb3e4ba4841.png">

7. Please try steps 3 & 4 again, and if they work, testing is complete!
